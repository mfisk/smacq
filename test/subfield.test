#!/bin/sh

bin/smacqq -m -O <<EOF 2>/dev/null
print -v, ipid.uint32.hexstring from pcapfile($TEST/0325@1112-snort.pcap)
EOF
