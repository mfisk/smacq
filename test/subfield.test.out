ipid.uint32.hexstrin = 0x145
ipid.uint32.hexstrin = 0x146
ipid.uint32.hexstrin = 0x147
ipid.uint32.hexstrin = 0x148
ipid.uint32.hexstrin = 0x149
ipid.uint32.hexstrin = 0x14a
ipid.uint32.hexstrin = 0x14b
ipid.uint32.hexstrin = 0x14c
ipid.uint32.hexstrin = 0x14d
ipid.uint32.hexstrin = 0x14e
ipid.uint32.hexstrin = 0x14f
ipid.uint32.hexstrin = 0x150
ipid.uint32.hexstrin = 0x151
ipid.uint32.hexstrin = 0x152
ipid.uint32.hexstrin = 0x153
ipid.uint32.hexstrin = 0x154
ipid.uint32.hexstrin = 0x155
ipid.uint32.hexstrin = 0x156
ipid.uint32.hexstrin = 0x157
ipid.uint32.hexstrin = 0x158
ipid.uint32.hexstrin = 0x159
ipid.uint32.hexstrin = 0x15a
ipid.uint32.hexstrin = 0x15b
ipid.uint32.hexstrin = 0x15c
ipid.uint32.hexstrin = 0x15d
ipid.uint32.hexstrin = 0x15e
ipid.uint32.hexstrin = 0x15f
ipid.uint32.hexstrin = 0x160
ipid.uint32.hexstrin = 0x161
ipid.uint32.hexstrin = 0x162
ipid.uint32.hexstrin = 0x163
ipid.uint32.hexstrin = 0x164
ipid.uint32.hexstrin = 0x165
ipid.uint32.hexstrin = 0x166
ipid.uint32.hexstrin = 0x167
ipid.uint32.hexstrin = 0x168
ipid.uint32.hexstrin = 0x169
ipid.uint32.hexstrin = 0x16a
ipid.uint32.hexstrin = 0x16b
ipid.uint32.hexstrin = 0x16c
ipid.uint32.hexstrin = 0x16d
ipid.uint32.hexstrin = 0x16e
ipid.uint32.hexstrin = 0x16f
ipid.uint32.hexstrin = 0x170
ipid.uint32.hexstrin = 0x171
ipid.uint32.hexstrin = 0x172
ipid.uint32.hexstrin = 0x173
ipid.uint32.hexstrin = 0x174
ipid.uint32.hexstrin = 0x175
ipid.uint32.hexstrin = 0x176
ipid.uint32.hexstrin = 0x177
ipid.uint32.hexstrin = 0x178
ipid.uint32.hexstrin = 0x179
ipid.uint32.hexstrin = 0x17a
ipid.uint32.hexstrin = 0x17b
ipid.uint32.hexstrin = 0x17c
ipid.uint32.hexstrin = 0x17d
ipid.uint32.hexstrin = 0x17e
ipid.uint32.hexstrin = 0x17f
ipid.uint32.hexstrin = 0x180
ipid.uint32.hexstrin = 0x181
ipid.uint32.hexstrin = 0x182
ipid.uint32.hexstrin = 0x183
ipid.uint32.hexstrin = 0x184
ipid.uint32.hexstrin = 0x185
ipid.uint32.hexstrin = 0x186
ipid.uint32.hexstrin = 0x187
ipid.uint32.hexstrin = 0x188
ipid.uint32.hexstrin = 0x189
ipid.uint32.hexstrin = 0x18a
ipid.uint32.hexstrin = 0x18b
ipid.uint32.hexstrin = 0x18c
ipid.uint32.hexstrin = 0x18d
ipid.uint32.hexstrin = 0x18e
ipid.uint32.hexstrin = 0x18f
ipid.uint32.hexstrin = 0x190
ipid.uint32.hexstrin = 0x191
ipid.uint32.hexstrin = 0x192
ipid.uint32.hexstrin = 0x193
ipid.uint32.hexstrin = 0x194
ipid.uint32.hexstrin = 0x195
ipid.uint32.hexstrin = 0x196
ipid.uint32.hexstrin = 0x197
ipid.uint32.hexstrin = 0x198
ipid.uint32.hexstrin = 0x199
ipid.uint32.hexstrin = 0x19a
ipid.uint32.hexstrin = 0x19b
ipid.uint32.hexstrin = 0x19c
ipid.uint32.hexstrin = 0x19d
ipid.uint32.hexstrin = 0x19e
ipid.uint32.hexstrin = 0x19f
ipid.uint32.hexstrin = 0x1a0
ipid.uint32.hexstrin = 0x1a1
ipid.uint32.hexstrin = 0x1a2
ipid.uint32.hexstrin = 0x1a3
ipid.uint32.hexstrin = 0x1a4
ipid.uint32.hexstrin = 0x1a5
ipid.uint32.hexstrin = 0x1a6
ipid.uint32.hexstrin = 0xa0e5
ipid.uint32.hexstrin = 0x1a7
ipid.uint32.hexstrin = 0x1a8
ipid.uint32.hexstrin = 0x1a9
ipid.uint32.hexstrin = 0x1aa
ipid.uint32.hexstrin = 0x1ab
ipid.uint32.hexstrin = 0x1ac
ipid.uint32.hexstrin = 0x1ad
ipid.uint32.hexstrin = 0x1ae
ipid.uint32.hexstrin = 0x1af
